gulpfile.js

```diff
@@ -40,10 +40,13 @@ function buildStatic () {
           main: './main.js',
           // Styles entry points
           'styles/main': './styles/main.styl',
+          // TODO: Ideally we want this to be a wildcard match of some form
+          // so we can automatically pick-up new custom styles.
+          'styles/custom/gamegig2017': './styles/custom/gamegig2017.styl'
```

Comment:
Could do with an issue for this

Reply:
Is there a GitHub thing called a `debate`? ;)

Reply:
Created #173 for this.